ansible/playbooks/teleop_deploy.yml

```diff
@@ -162,6 +162,11 @@
 - name: Install Teleop software on control machine
   hosts: control_machine
   pre_tasks:
+    - name: Deploy conda on control machine
+      include_role:
+        name: installation/conda_deploy
+      # when: not skip_molecule_task|bool
```

Comment:
why is this line left commented?

Reply:
thanks 72d8a5f9a8a410748241eddc2e9c45998f1b7d76